src/test/ui/feature-gates/feature-gate-raw-dylib-2.rs

```diff
@@ -0,0 +1,8 @@
+#[link(name="foo")]
```

Comment:
Please move this and the other feature-gate file to `src/test/ui/rfc-2627-raw-dylib/`. More test files should be added there over time.